node_modules/.package-lock.json

```diff
@@ -0,0 +1,2152 @@
+{
```

Comment:
Since the project packages are installed into this folder, it would be good to keep this folder local rather than on the Github.  `node_modules`